package.json

```diff
@@ -40,9 +40,9 @@
     "karma-firefox-launcher": "~1.1.0",
     "karma-mocha": "^2.0.1",
     "karma-sinon": "^1.0.5",
-    "mocha": "^8.0.1",
+    "mocha": "^8.3.1",
     "sinon": "^7.2.3",
-    "webpack": "~4.27.1",
-    "webpack-cli": "^3.3.11"
+    "webpack": "^4.46.0",
```

Comment:
This is the latest release in the webpack v4 series. There is a webpack v5 series, but upgrading to it seems to generate a suspiciously big diff in `dist/FxAccountsPairingChannel.js` so I don't think we should do that right now.